codespell_lib/data/dictionary.txt

```diff
@@ -17963,6 +17963,7 @@ instutions->intuitions
 insue->ensue, insure,
 insufficent->insufficient
 insuffiency->insufficiency
+insuffient->insufficient
```

Comment:
```suggestion
insufficency->insufficiency
insufficent->insufficient
insufficently->insufficiently
insuffiency->insufficiency
insuffient->insufficient
insuffiently->insufficiently
```

Reply:
* https://github.com/search?q=insufficency&type=code  393
* https://github.com/search?q=insufficently&type=code  620
* https://github.com/search?q=insuffient&type=code  66452
* https://github.com/search?q=insuffiently&type=code  208

You might also want to add the following as well.
```text
sufficency->sufficiency
suffiency->sufficiency
suffient->sufficient
suffiently->sufficiently
```
* https://github.com/search?q=sufficency&type=code  1206
* https://github.com/search?q=suffiency&type=code  666
* https://github.com/search?q=suffient&type=code  286066
* https://github.com/search?q=suffiently&type=code  35388